src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -39,6 +40,8 @@ internal TagHelperAttributeDescriptor([NotNull] string name, [NotNull] PropertyI
         /// If <c>true</c> this <see cref="TagHelperAttributeDescriptor"/> is used for dictionary indexer assignments.
         /// Otherwise this <see cref="TagHelperAttributeDescriptor"/> is used for property assignment.
         /// </param>
+        /// <param name="usageDescriptor">The <see cref="TagHelperUsageDescriptor"/> that indicates how to use the
+        /// tag helper attribute during design time.</param>
```

Comment:
No indications here and "during design time" is not worth mentioning. For all documentation of `UsageDescriptor` parameters and properties, say `... that contains information about use of this attribute.` or `... this tag helper`. If you want to add `<remarks>` mentioning the properties are `null` at runtime, I guess that's OK but that ignores all the other cases in which the descriptor will be `null`.